app/views/warehouse_reports/chronic/_homeless_info.html.haml

```diff
@@ -0,0 +1,25 @@
+.block-highlight
```

Comment:
@eanders - i moved this to the bottom of page due to sorting causing a page refresh which then causes the user to have to scroll to see the results. Let me know if this is ok. If not, i can change to an ajax request and move this back to the top.